index.html

```diff
@@ -125,10 +132,10 @@ <h2 class="title">Tonic</h2>
                     </div>                
             </article>
 <!--card_4 starts from here-->
```

Comment:
- [ ] Cards are almost there! However, the 2nd and 4th cards are not being displayed properly, there is a blank space taking half of the screen, the height is different from the other cards and the overall alignment is fairly different from the Figma template, let's try to make it more similar to the guidelines:

|Template|Your design|
|---|---|
|![Captura de pantalla 2022-06-02 a la(s) 10 38 55 p  m](https://user-images.githubusercontent.com/83717285/171783667-a5d078fb-32f1-4ae8-af3c-ff559d9c26b5.png)|![Captura de pantalla 2022-06-02 a la(s) 10 39 06 p  m](https://user-images.githubusercontent.com/83717285/171783776-7be09291-2dd5-414d-b0fa-a7349d1aec7d.png)|

Reply:
This issue is still persisting, it might be because your image is set to a fixed small width. But your image is too small. Here is a reference. Also kindly make the size of the cards bigger as per your template.
![image](https://user-images.githubusercontent.com/33268927/171985014-7642c8bc-99f1-4c93-bed4-08e376ffc3e5.png)